errors/standardErrors.ts

```diff
@@ -3,21 +3,28 @@ import { i18n } from '../utils/lang';
 
 import { BaseError } from '../types/Error';
 import { LangKey } from '../types/Lang';
-import { AxiosError } from 'axios';
 
-export function isSystemError(err: BaseError): boolean {
-  return err.errno != null && err.code != null && err.syscall != null;
+export function isSystemError(err: unknown): err is BaseError {
+  return (
+    err instanceof Error &&
+    'errno' in err &&
+    err.errno != null &&
+    'code' in err &&
+    err.code != null &&
+    'syscall' in err &&
+    err.syscall != null
+  );
 }
 
-export function isFatalError(err: BaseError): boolean {
+export function isFatalError(err: unknown): err is BaseError {
```

Comment:
Should this be `err is HubspotAuthError`?

Reply:
Also realizing that this is redundant with `isHubSpotAuthError`, but I guess we can't really remove it without doing a major version bump

Reply:
Great question.  My reasoning for going `BaseError` over `HubSpotAuthError` was that this method might expand to cover other types of errors so I wanted to keep it generic.

Reply:
Maybe we can remove `isFatalError` in one of your follow up PR's? I'm guessing this rework of the error handling will eventually require a major version bump anyways.

Reply:
> Maybe we can remove isFatalError in one of your follow up PR's?

Yep, I will do this in the batch of PRs after the [AxiosError](https://github.com/HubSpot/hubspot-local-dev-lib/pull/151/files) rework